src/localization/buttonStrings.json

```diff
@@ -68,8 +68,8 @@
     "realm_explorer": "Explorateur Realm",
     "save_changes": "Sauvegarder les modifications",
     "settings": "RÉGLAGES",
-    "use_requested_quantities": "Utiliser qté requises",
-    "use_suggested_quantities": "Utiliser qté suggérées",
+    "use_requested_quantities": "Utiliser quantité requise",
```

Comment:
This one looks to be cut off due to the length (whereas Utiliser qté requises just seems to fit)

Reply:
Fixed by abbreviating (and showing abbreviation) in [5835e67](https://github.com/openmsupply/mobile/pull/4648/commits/5835e67eac320503c79269ae97d6bf50402bd555) and [221735c](https://github.com/openmsupply/mobile/pull/4648/commits/221735ca69eed884f6b340cffa622b0b0b866d06)